packages/flutter_migrate/lib/src/base/terminal.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:async';
```

Comment:
why add this without any other new code?

Reply:
This was leftover from some experiments. Missed it in cleanup

Reply:
Interesting that the analyzer didn't catch this

Reply:
yeah...